nerdlandbot/commands/space_launches.py

```diff
@@ -0,0 +1,117 @@
+import aiohttp
+import asyncio
+import json
+import os
+import discord
+from datetime import datetime
+from discord.ext import commands
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+
+from nerdlandbot.helpers.constants import \
+                                        THE_SPACE_DEVS_BASE_URL, THE_SPACE_DEVS_VERSION, THE_SPACE_DEVS_LIMIT_TO_10_RESULTS, \
+                                        THE_SPACE_DEVS_UPCOMING_LAUNCH_RESOURCE, \
+                                        THE_SPACE_DEVS_LOCAL_TEST_SERVER_BASE_URL, \
+                                        THE_SPACE_DEVS_LOCAL_CACHE_FOLDER, THE_SPACE_DEVS_LOCAL_CACHE_SPACE_LAUNCHES_FILE, \
+                                        THE_SPACE_DEVS_HOME_URL, THE_SPACE_DEVS_TIMESTAMP_FORMAT
+
+class SpaceDevs (commands.Cog, name='The space devs'):
+    def __init__(self,bot:commands.Bot):
+        self.bot = bot
+        self.ctx = None
+
+    @commands.command(name="space_launches", hidden = False, help="space_launches_help", brief="space_launches_brief", usage="space_launches_usage")
+    async def cmd_space_launches(self, ctx:commands.Context):
+        self.ctx = ctx
+#        full_url = '/'.join ([THE_SPACE_DEVS_LOCAL_TEST_SERVER_BASE_URL, THE_SPACE_DEVS_LOCAL_CACHE_SPACE_LAUNCHES_FILE + '.json'])
+        full_url = '/'.join ([THE_SPACE_DEVS_BASE_URL, THE_SPACE_DEVS_VERSION, '/'.join (THE_SPACE_DEVS_UPCOMING_LAUNCH_RESOURCE)])
+
+        if self.should_call_the_api ():
+            async with aiohttp.ClientSession() as session:
+                async with session.get(full_url, headers = {"accept":"application/json"}) as resp: 
+                    if resp.status == 200:
+                        msg = await resp.text()
+                        with open(os.path.join(THE_SPACE_DEVS_LOCAL_CACHE_FOLDER, THE_SPACE_DEVS_LOCAL_CACHE_SPACE_LAUNCHES_FILE + '.json'),"w") as file:
+                            file.write(msg)
```

Comment:
If you use `json.dump(msg, indent=4)` here, the json file will be separated in multiple lines instead of one very long line

Reply:
This is a caching file which will only needs to be checked in case there are problems.
Every byte counts :-)

Reply:
But in case there are problems, it will be a lot easier to check this file if it is not one very long line? How many bytes are you expecting that it will make that much of a difference? 
Anyway, this is just a comment, I won't block the merge for this, but I would prefer reading a separated file much more if there are problems